fe1-web/parts/Wallet.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useState } from 'react';
```

Comment:
I've pointed out a few issues, but this file to me seems particularly messy.
It also seems to contain both demo-specific code and actually useful things.
You might want to clean it up:
1. If you need navigation, use React navigation, not your own state machine and conditional display logic.
2. If code is only for demo, maybe split it out in a file that can be easily removed later.
3. If the code is actually useful in the long run, keep it clean.
4. Make sure that variables are properly named, I would recommend reading [Code Complete, Chapter 11, PDF page 296 and following](http://aroma.vn/web/wp-content/uploads/2016/11/code-complete-2nd-edition-v413hav.pdf) in this particular case, and the whole book to any programmer anyways 😄